scripts/generate_duration.py

```
@@ -14,50 +20,63 @@
+                    # Fall back to last column if the fixed index doesn't work
+                    try:
+                        duration = float(row[-1].strip())
+                    except ValueError:
+                        print(
+                            f"  Warning: Could not parse duration for {test_name}. Skipping."
+                        )
+                        continue
+
+                test_durations[test_name] = duration
+                passed_count += 1
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)